### DISCUSSION TITLE
Siovox bug?

### DISCUSSION THREAD

**Discussion by omgjkvyufui** [3kh0/ext-remover#903] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
I have done all the steps to the SKIOVOX procces but once i trun on wifi can close it there is no unblocked tab behind idk if im doing smth wrong

**Comment by CGRmags** [comment]
@omgjkvyufui What Version are you on? because you should consider downgrading if you are on 120-119 because I think It only works on 118 or below! Someone say smth if I am wrong

**Reply by omgjkvyufui** [reply]
Im on 120 and downgrading is also not available on 120

**Reply by v4pl** [reply, marked as answer]
I use 120 and it works just use the new method.

**Comment by v4pl** [comment]
Does work on Verson 120 for me.
